test/Microsoft.Sbom.Parsers.Spdx30SbomParser.Tests/Parser/JsonStrings/SbomFullDocWithPackagesStrings.cs

```diff
@@ -0,0 +1,716 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.Sbom.Parser.JsonStrings;
+
+public static class SbomFullDocWithPackagesStrings
+{
+    [System.Diagnostics.CodeAnalysis.SuppressMessage("Style", "JSON002:Probable JSON string detected", Justification = "Need to use JSON string")]
+    public const string SbomPackageWithMissingVerificationJsonString =
+    @"
```

Comment:
During my DRI week, I may try to convert all of our inline json files to actual on-disk json files, both to make the tests smaller and to get rid of the double-quotes